cloudproxy/providers/config.py

```diff
@@ -13,5 +13,5 @@ def set_auth(username, password, allowed_ip):
         else:
             filedata = filedata.replace("BasicAuth username password", "#BasicAuth username password")
         if len(allowed_ip) > 0:
-            filedata = filedata.replace("#Allow 127.0.0.1", "Allow " + allowed_ip)
+            filedata = filedata.replace("#Allow 127.0.0.1", "Allow " + allowed_ip + "\/24")
```

Comment:
My understanding here is your adding /24 to the end of the allowed_ip which would mean the entire subnet has access. Not sure this is very good idea and not sure the reasoning behind it. A total of 256 IPs would then have access, which another server in the same subnet in the cloud provider could then exploit, if they were looking.